.github/workflows/ci.yml

```diff
@@ -57,14 +57,23 @@ jobs:
         run: |
           go build -o runme main.go
           ./runme --version
+      - name: Test All
```

Comment:
This is duplication of what's below (`name: Test`) but without coverage. It runs exactly the same set of tests.

We can argue if it should be `TAGS="test_with_docker" make test-coverage` (notice instead of `-` which is a different Makefile target) in the step below to bring it closer to `make test`.